nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config

```diff
@@ -34,6 +34,11 @@ params {
   config_profile_contact = false
   config_profile_url = false
 
+  // AWSBatch
```

Comment:
I don't think this should be in the template, but in the nf-core/configs instead

Reply:
How do you mean? The parameters will need to be specified and therefore initialised at the pipeline level.

Reply:
Maybe Im missing something about parameter propagation through configs 😅

Reply:
I mean here: https://github.com/nf-core/configs/blob/master/conf/awsbatch.config